cli/deploy-rest.sh

```diff
@@ -1,13 +1,15 @@
 ## IMPORTANT: this file and accompanying assets are the source for snippets in https://docs.microsoft.com/azure/machine-learning! 
 ## Please reach out to the Azure ML docs & samples team before before editing for the first time.
 
+set -x
+
 # <create_variables>
 SUBSCRIPTION_ID=$(az account show --query id | tr -d '\r"')
-LOCATION=$(az group show --query location | tr -d '\r"')
+LOCATION=eastus2euap
```

Comment:
can we remove hardcoding?

Reply:
then we need to query workspace for location, or ws should be located in the same place as RG